### ISSUE TITLE
Set language of speech synthesis (Linux)

### ISSUE THREAD

**Issue by Elinvention** [mumble-voip/mumble#2066]
Currently Mumble's speech synthesiser always use English and there is no way to change it. It reads any text with a very English accent. I think there should be a setting defaulting to the user's locale.
See also [Debian bug 600276](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=600276).

**Comment by mkrautz** [comment]
I'll give you what info I can.

	The bug is marked "upstream" but I can't find a bug in the Github bug 
	tracker for Mumble related to this.  Mumble uses libspeechd2; I had a look 
	at libspeechd.h in the libspeechd-dev package -- there's a public function 
	spd_set_language() which is what I would think would need to be called in 
	order to change the language spoken -- and the text "spd_set_language" does 
	not appear in the mumble source code.

	So based on this I don't think mumble is currently capable of switching 
	speech languages, let alone based on translation.  I also had a quick look 
	at the mumble git repo for spd_set_language -- same there too... so this 
	doesn't appear to be an upcoming feature either, AFAICT.

	I wish I had better news.

	 -- Chris

	-- 
	Chris Knadle
	Chris.Knadle@coredump.us

tl;dr

We should use spd_set_language().
